tokens/usecase/pricing/coingecko/pricing_coingecko_test.go

```diff
@@ -109,7 +109,7 @@ func (s *CoingeckoPricingTestSuite) TestGetPrices_Coingecko_FindUnsupportedToken
 
 	fmt.Println("Total unsupported tokens: ", unsupportedCounter)
 
-	// Total unsupported tokens as of May 24 2024: 136
-	s.Require().Equal(136, unsupportedCounter)
+	// Total unsupported tokens as of May 29 2024: 151
+	s.Require().Equal(151, unsupportedCounter)
```

Comment:
This is a sharp increase but should not be affected by the changed logic here.

This probably stems from mainnet state updates / asset listings.

Will run the test suite on this prior to merge

Reply:
I have checked this by only copying the test state update and observing the same number of breakages:
https://github.com/osmosis-labs/sqs/pull/273